### ISSUE TITLE
Allow user to have bookmarks-only side pane

### ISSUE THREAD

**Issue by MartinThoma** [mate-desktop/caja#480]
Currently, Caja looks like this for me:

![Imgur](http://i.imgur.com/fFamUaL.png)

As I've noted, I would like to be able to adjust the Side pane. Is this possible?

(P.S.: Other people seem to have the same issue - http://forums.linuxmint.com/viewtopic.php?t=137045&p=731132)

**Comment by raveit65** [comment]
All this depends  on personal point of view, for example i don't need bookmarks, so why should they on top?.
And changing defaults will always cause an issue by another user.
I agree that an option to make it variable is nice, but i don't think that is easy.

**Comment by MartinThoma** [comment]
Just to make sure: I'm not asking for a GUI way to customize this. I'm asking for a configuration file. Something like a file `~/.caja-places`:

```text
- {title: "Bookmarks",
   type: "org.mate-desktop.caja.sidepanepane.bookmarks"}
- {title: "Network",
   type: "org.mate-desktop.caja.sidepanepane.network"}
```

So a list of elements, where `title` is what is currently shown in bold and `type` is a list of identifiers known by caja. The file format in this example is YAML. I don't know if caja has any other configuration files. In case it has, one should stick with one configuration file format, of course. And I also don't really care about which format to take, as long as it is a standard / easy to understand / well documented format.

I don't care much about the default. I just don't want to reach where to adjust this in the MATE source and compile it myself.

**Comment by colibris79** [comment]
I agree that this would be a nice feature. I also like bookmarks on top as they are the most important, but that, obviously depends on personal preferences. Would be great to be able to configure it according to whatever each person wants.

**Comment by ulgrimar** [comment]
Me too. Would be very nice.

**Comment by fbarriga** [comment]
related with #160

**Comment by localstyle** [comment]
Same here.
2 reasons why the current layout is absurdly counter intuitive to me:

- Why is Filesystem stuck between Desktop and Documents? I mean, first bookmark is Home, alright. Then Desktop, alright, since it's the first folder in Home. Logically, Documents and so on should follow! Why is Filesystem there?? I want to move it above the Trash.

- The actual folders are ranked by name in the OS's language, whereas the bookmarks are ranked based on the system name (that is, in english) of the folder. Ex. my "Images" folder is referred to "PICTURES" in the system. That means - if you're language is not english and/or if you want to rename your default folders and bookmarks, the order in the left pane will be different than the one in the Home folder.

A config file where the order can be customized would be veeeeery welcome! :)

**Comment by raveit65** [comment]
If you split up the file-sytem in different partitions it makes sense to have it under home for me ;-)
I use about 10 partitions in my system, so i like this layout.
But i agree that it should be configurable.

**Comment by localstyle** [comment]
Of course, I'm not saying the current layout makes absolutely no sense. It doesn't to me, that's all. Different people, different needs, that's why it should be configurable. But the translation issue probably does affect a few million people...

**Comment by localstyle** [comment]
How about I just remove the defaults for Documents etc. by editing user-dirs.dirs, create my own bookmarks for those folders (solves bullet point 2 and pretty much bullet point 1 as well) and stop bitching? :)

**Comment by shirishag75** [comment]
I am running mate 1.20.0 and have no idea where caja puts it bookmarks. The bookmarks feature seems to be broken and is also not documented in caja help <F1> . The only sort of uglyish workaround I was able to figure out is install nautilus and bookmark it in the nautilus side-pane and caja would take the bookmarks from there. Although again where is all this data put is not known. I did cycle through all . directories but wasn't able to get the required info.

**Comment by raveit65** [comment]
You can create bookmarks with the bookmark menu.
Bookmarks are stored here
~/.config/gtk-3.0/bookmarks
No need to install nautilus....

**Comment by shirishag75** [comment]
@raveit65 that should have been documented and I did try to place the bookmarks using mouse but was unable to . 

Also how that should be isn't given. For e.g. - 

```
~/.config/gtk-3.0$ cat bookmarks 
file:///home/shirish/Video-songs/english-songs
file:///home/shirish/Video-songs/hindi-songs
file:///home/shirish/Video-songs
file:///home/shirish/Videos

```
